darts/tests/models/forecasting/test_torch_forecasting_model.py

```diff
@@ -290,6 +291,74 @@ def test_train_from_10_n_epochs_20_fit_15_epochs(self):
             model1.fit(self.series, epochs=15)
             self.assertEqual(15, model1.epochs_trained)
 
+        def test_load_weights_from_checkpoint(self):
+            original_model_name = "original"
+            retrained_model_name = "retrained"
+            # original model, checkpoints are saved
+            model1 = RNNModel(
+                12,
+                "RNN",
+                5,
+                1,
+                n_epochs=5,
+                work_dir=self.temp_work_dir,
+                save_checkpoints=True,
+                model_name=original_model_name,
```

Comment:
we should fix the `random_state` with these tests, as there is a possibility that they will fail at some point